src/Rings/mpoly.jl

```diff
@@ -1147,3 +1147,20 @@ end
 function _is_integral_domain(R::MPolyRing) 
   return true
 end
+
+@doc Markdown.doc"""
+    total_degree(f::MPolyRingElem, w::Vector{Int})
```

Comment:
total_degree for polynomials without specified weight exists somewhere else. Wouldn't it be a good idea to have both methods in the same place?

Reply:
So far we had two identfiers here: total_degree = the usual total degree.
                                                        degree = degree of an element of a graded polynomial ring
Both are in accordance with the use in the literature and are introduced in the docu. 

Question to everybody: Shouldn`t we use something like weighted_degree here and stick to total_degree as it was?

Reply:
I agree, weighted_degree avoids ambiguities, is in accordance with the use in literature (in particular in the weighted local setting for (semi-)quasihomogeneous singularities) and is a logical choice, when users look for it with <Tab>-completion.

Reply:
I'm fine with calling it whatever name you find suitable. Let me know when you reach a conclusion.

Reply:
We seem to agree on `weighted_degree`.